Sources/NIO/ByteBuffer-core.swift

```diff
@@ -235,7 +235,7 @@ public struct ByteBuffer {
             return self.allocateStorage(capacity: self.capacity)
         }
 
-        private func allocateStorage(capacity: _Capacity) -> _Storage {
+        public func allocateStorage(capacity: _Capacity) -> _Storage {
```

Comment:
This should be `internal`: it doesn’t need higher visibility than the containing object.

Reply:
Sure! Updated!